content/en/security/cloud_workload_security/security_profiles.md

```diff
@@ -0,0 +1,78 @@
+---
+title: CWS Workload Security Profiles
+kind: documentation
+further_reading:
+  - link: "/security/cloud_workload_security/setup"
+    tag: "Documentation"
+    text: "Setting Up CWS"
+  - link: "/security/cloud_workload_security/workload_security_rules"
+    tag: "Documentation"
+    text: "Managing Cloud Workload Security Rules"
+---
+
+Workload Security Profiles provide a baseline of expected workload activity via a behavioral learning model that helps identify potential threats or misconfigurations. This insight can be used when investigating security alerts, including [generating suppression suggestions](#suppress-signals-based-on-suggestions) for known, acceptable workload behavior, as well as identifying previously unseen, anomalous behavior.
+
+## Compatibility
+
+For Workload Security Profiles to be compatible with your Datadog configuration, you must have [CWS enabled][1], use Datadog Agent 7.44 or later, and have activity dumps enabled. Workload Security Profiles are enabled by default and support containerized workloads only.
+
+## How Workload Security Profiles are generated
+
+Workload Security Profiles are generated when [activity snapshots](#activity-snapshots) from containers sharing a common image name and image tag (version) are merged together to form a security profile. A different security profile is generated for each version of a container image. As a result, one profile is generated per `{ image-name, image-version}` tag combination.
+
+Each security profile acts as a [behavior model](#behavior-learning-model) for a given workload image to identify known, acceptable behavior that is expected to be consistent across all containers versus previously unseen, anomalous behavior that may be unique to a single container or small number of containers.
```

Comment:
```suggestion
Each security profile acts as a [behavior model](#behavior-learning-model) for a given workload image to identify known, acceptable behavior. The security profile uses the behavior model as a template for expected behavior that is consistent across all containers.  
```
Is this sentence a comparison? Before this product, behavior models would group on anomalous behavior? 
`previously, unseen, anomalous behavior that may be unique to a single container or small number of containers.`

Reply:
It's a comparison, but I see your point. It's more concise and accurate without the comparison.